src/notation/view/notationcontextmenumodel.cpp

```diff
@@ -200,6 +202,40 @@ MenuItemList NotationContextMenuModel::makeElementItems()
         items << makeMenuItem("edit-element");
     }
 
+    items << makeSeparator();
+
+    MenuItem* item = new MenuItem(this);
+
+    mu::ui::UiAction action;
+    action.code = "edit-style";
+    action.title = TranslatableString("notation", "Style...");
+    item->setAction(action);
+
+    QString id = QString::fromStdString(item->action().code) + QString::number(0);
+    item->setId(id);
+
+    mu::ui::UiActionState state;
+    state.enabled = true;
+    item->setState(state);
```

Comment:
You also should use uiActionsRegister()->actionState(actionCode) to get the actual action state